doc/explanations/testing-and-debugging.md

```diff
@@ -0,0 +1,76 @@
+Robotoff is an API that pulls prediction data, annotation data, product data, nutrition data from MongoDB database in Open Food Facts server.
+
+
+# This documentation intends to:
+1) Familiarize you on running our test cases.
+2) Give you some tips from getting stuck if you are working only on Robotoff and don't need complete installation of Open Food Facts server.
+
+
+> **_NOTE:_** If you are on Windows we recommend using [Git bash](https://git-scm.com/downloads) to run commands.
+
+# How to generate data
+We recommend  [Factory](https://factoryboy.readthedocs.io/en/stable/) to create some data in your local.
+
+If you have installed Robotoff via Docker, you can run Python using Poetry and execute Factory like so:
+```
+$ docker-compose run --rm api poetry run python
```

Comment:
`docker-compose run --rm api poetry run python` only starts the api for me and does not give me a console to run the commands. Since you had once suggested me to use this in our Slack conversation, I have added in the document.

@alexgarel  let me know if I should keep or remove the command.